src/components/Navbar/NavTabs.js

```diff
@@ -7,64 +7,91 @@ import Logo from  "./Logo"
 import "./NavAccessibility.css"
 
 
-const useStyles = makeStyles({
-   root: {
-            margin: "0px",
-            padding: "0px",
-            boxSizing: "border-box"
-        },
-  navbar: {
-            position: "sticky",
-            left: "0%",
-            right: "0%",
-            top: "0%",
-            height: "80px",
-            background: "#FFFFFF",
-            boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
-            padding: "0 8.3%",
-            color: "black",
-            display: "flex",
-            flexWrap: "wrap",
-            alignItems: "center",
-            justifyContent: "space-between",
-            fontFamily: 'Poppins'
-        },
-  navtabs: {
-            width: "fit-content",
-            height: "80px",
-            float: "right!important",
-            display: "flex",
-            flexWrap: "wrap",
-            paddingLeft: "0",
-            marginBottom: "0",
-            listStyle: "none",
-            marginTop: "0"
-        },
-  navitem: {
-            marginBottom: "-1px"
-  },
-
-      navlink: {
-            width: "104px",
-            fontSize: "16px",
-            textAlign: "center",
-            lineHeight: "80px",
-            color: "black",
-            padding: "0px",
-            display: "block",
-            textDecoration: "none",
-  },
-         active: {
-            boxShadow: "0px 2px 0px #873FE2",
-            lineHeight: "80px",
-            color: "#873FE2!important"
-        }
-
-});
-
-
+/** sizing  navbar padding based on window width**/
+ 
+
+  function useWindowSize() {
+    const [windowSize, setWindowSize] = useState({
+      width: undefined,
+    });
+    
+    useEffect(() => {
+      // Handler to call on window resize
+      function handleResize() {
+        // Set window width/height to state
+        setWindowSize({
+          width: window.innerWidth,
+        });
+      }
+      // Add event listener
+      window.addEventListener("resize", handleResize);
+    
+      // Call handler right away so state gets updated with initial window size
+      handleResize();
+
+      // Remove event listener on cleanup
+      return () => window.removeEventListener("resize", handleResize);
+    }, []); // Empty array ensures that effect is only run on mount
+
+    // The padding value as a function of screen size
+    let navbarPadding,navbarHeight, navitemWidth, logoWidth, logoFontSize; 
+    let buttonHeight, buttonWidth, heartFontSize, heartWidth;
+    let navtabsDisplay, burgerMenuDisplay;
+    
+    if (windowSize.width >= 1040) {
+      navbarPadding = (windowSize.width / 4 - 240).toFixed(1);
```

Comment:
Sounds like coding is done dynamically and nicely, great job @BBoutet1 !

I have some queries for my understanding: 
1) what is the formula/logic used for deriving these values? I mean how did you decide "divide by 4" and then reduce by "240" types?
2) And the logoWidth, logoFontSize etc values are from Figma I take?

Reply:
1. I took the values used in figma for 360px, 768px and 1440px. The challenge was to maintain the tabs aligned while the screen size is reducing. In that case I needed to reduce the size progressively. I calculated the formulas (y=ax+b) to stay close to figma values at 360, 768 and 1440 px for the navbar margin, tabs and button size. 2. Yes, I follow figma for width, height and font sizes.

Reply:
ok, and the coefficients a & b were derived using hit-&-trial? And I cannot figure out why you would choose a line equation, can you share that reason separately too (just for my curiosity)

sidenote: for y=ax+b, dont you want to then update ur formula to x=(y-b)/a (instead of x=y/a-b)?

Am sorry I dont have any other inputs/contributions for this review as of now (not able to take out time to study react/material-ui)

Reply:
Any equation that allows progressive width reduction while maintaining tabs alignment and keeping figma values at specified break points is welcome. Line equation is the simplest. (y,x) are the element width and the screen size.

Reply:
@harshvchawla  Will give you more details on slack.

Reply:
thanks @BBoutet1